src/js/themes/base.js

```diff
@@ -345,6 +345,14 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         previous: Previous,
         // color: undefined,
       },
+      animation: {
+        duration: 1000,
+      },
+      disabled: {
+        icons: {
+          color: undefined,
```

Comment:
To be consistent with the rest of the file, can we comment out the undefined value? (e.g line 346)